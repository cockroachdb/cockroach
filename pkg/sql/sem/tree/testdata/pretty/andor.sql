select a and (b or c) and d,
  a or (b and c) or d
