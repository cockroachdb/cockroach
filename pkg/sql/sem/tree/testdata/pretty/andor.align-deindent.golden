// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	a
	AND (
			b
			OR c
		)
	AND d,
	a
	OR (
			b
			AND c
		)
	OR d

8:
--------
SELECT a
   AND (
		b
		OR c
       )
   AND d,
       a
    OR (
		b
		AND c
       )
    OR d

15:
---------------
SELECT a
   AND (b OR c)
   AND d,
       a
    OR (
		b
		AND c
       )
    OR d

16:
----------------
SELECT a
   AND (b OR c)
   AND d,
       a
    OR (b AND c)
    OR d

48:
------------------------------------------------
SELECT a AND (b OR c) AND d, a OR (b AND c) OR d


